native/Avalonia.Native/src/OSX/AvnView.mm

```diff
@@ -290,6 +290,39 @@ - (void)mouseEvent:(NSEvent *)event withType:(AvnRawMouseEventType) type
         delta.Y = [event deltaY];
     }
 
+    float pressure = 0.0f;
```

Comment:
Sorry, I forgot about this one, that should be also set to 0.5 to keep the previous behavior for mouse, in case people are relying on it.